client/client.go

```diff
@@ -27,17 +28,19 @@ func NewClient(addr, tmsp string) (*Client, error) {
 }
 
 func NewLocalClient() *Client {
-	eyesApp := app.NewMerkleEyesApp()
+	eyesApp := app.NewMerkleEyesApp("", 0) //TODO does this make sense? create a non-persistent instance if a local client?
```

Comment:
Should we add persistence to NewLocalClient? Right now this is initialized by default as non-persistent.